CIP-0XXX/README.md

```diff
@@ -0,0 +1,551 @@
+socket. Since `Mithril` nodes will have their own end-points, we'll either need
+to propose a modification to the SPO registration process, which includes
+`Mithril` nodes, or we could pass incoming `mithril` connections from
+`cardano-node` to the `Mithril` node via
+[CMSG_DATA](https://man.openbsd.org/CMSG_DATA.3).
+
+`Ouroboros-Network` outbound governor component, which is responsible for
+creating the overlay network has a built-in mechanism which churns connections
+based on a defined metric.  By developing a standalone `Mithril` node, we can
+design metrics specific for the purpose.  This way, the `Mithril` network will
+optimise for its benefits rather than being stuck in a suboptimal solution from
+its perspective - if `Mithril` and `Cardano` were tied more strongly (e.g. as
+part `cardano-node`), then we wouldn't have that opportunity because `Cardano`
+must meet its deadlines, or the security of the system as a whole must be at
+stake.
+
+Eventually, there will be many `Mithril` applications, and all of them might
+have different network requirements and thus might require slightly different
+configurations.  We SHOULD aim to build something that can be used as
+a scaffolding for such applications. This may also open future avenues for
+delivering new functionalities that fit together well with existing
+infrastructure while still mitigating the risk of catastrophic events at the
+core of the system.
+
+Please also note that any protocols and their instances that will be built as
+part of the standalone `Mithril` node could be reused in future for `Peras` and
+`Leios`.  It will give us even more confidence that the future core system will
+be built from components that are used in production.
+
+`Cardano` and `Mithril`, as separate executables, can still be packaged
+together, lowering the barrier of participation. When run separately, the SPO
+is also in better control of the resources dedicated to each process - this is
+important for the health of both systems.
+
+Another benefit of such a design is that `Mithril` can be developed on its own
+pace, not affected by significant changes in other parts of the system like
+ledger - the Cardano Core team restrains itself to not publishing new `cardano-node`
+versions with significant changes across many of its subsystems - just for the
+sake of clarity of where to look if a bug is found.
+
+In this design, a `Mithril` node runs alongside a `cardano-node`, which is connected to
+it via a UNIX socket (`node-to-client`) protocol.  This means an SPO will run
+a `Mithril` node per `cardano-node`.  The `Mithril` node connected to BP can
+also have access to necessary keys for signing purposes. `Mithril` might also
+use the KES agent, as `cardano-node` will in the near future, to securely access
+the KES key.
+
+
+# Path to Active
+
+## Acceptance Criteria
+
+1. A Cardano node implementing the previously described mini-protocols is released for production.
```

Comment:
I don't think the wording is correct here. It should not be "a cardano node" but rather "a mithril node", right?